rn.sh

```diff
@@ -0,0 +1,20 @@
+cd ./release
+mv bitcoind vbitcoind
+mv bitcoin-cli vbitcoin-cli
+mv bitcoin-qt vbitcoin-qt
+mv bitcoin-tx vbitcoin-tx
+mv bitcoind.exe vbitcoind.exe
+mv bitcoin-cli.exe vbitcoin-cli.exe
+mv bitcoin-qt.exe vbitcoin-qt.exe
+mv bitcoin-tx.exe vbitcoin-tx.exe
+cd ..
+cd ./src
+mv bitcoind vbitcoind
+mv bitcoin-cli vbitcoin-cli
+mv qt/bitcoin-qt qt/vbitcoin-qt
+mv bitcoin-tx vbitcoin-tx
+mv bitcoind.exe vbitcoind.exe
+mv bitcoin-cli.exe vbitcoin-cli.exe
+mv qt/bitcoin-qt.exe qt/vbitcoin-qt.exe
+mv bitcoin-tx.exe vbitcoin-tx.exe
```

Comment:
I think would be easier to change name of targets once in `src/Makefile.am` rather than adding bash script. Windows users, may not have bash, for example

Reply:
My bad. This was a local file which sneaked through and didn't pull down the deletion when I did my git pull (it was trimmed prior so it was still local - fixed.)